build_project/local.go

```diff
@@ -0,0 +1,66 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func buildProjectLocally(projectCtx project.ProjectCtx) error {
+	// pre-build
```

Comment:
But where is post-build hooks?

Reply:
Post build hook runs only on packing (it removes some files that shouldn't be delivered in a result artifact, but we shouldn't remove anything on building in the current directory).